docs/source/coroutines.rst

```diff
@@ -50,6 +50,52 @@ Coroutines can :keyword:`return` a value, so that they can be used by other coro
         second = await get_signal(dut.clk, dut.signal)
         assert first != second, "Signal did not change"
 
+
+Useful Objects for Testbenches
```

Comment:
This is very generic. Maybe "Triggers and Concurrency Utilities"? Same with the sentence below.